src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -59,6 +59,10 @@ public static class EnvironmentSettingNames
         public const string TestDataCapEnabled = "WEBSITE_FUNCTIONS_TESTDATA_CAP_ENABLED";
         public const string AzureMonitorCategories = "WEBSITE_FUNCTIONS_AZUREMONITOR_CATEGORIES";
 
+        //Function in Kubernetes
+        public const string PodNamespace = "POD_NAMESPACE";
+        public const string KubernetesAppServiceNamespace = "k8se-apps";
```

Comment:
This might be a problem as it seems that we might not have control over the namespace the user deploys apps into.

Reply:
In that case I guess, it might be sufficient to check non-empty or null value of POD_NAMESPACE? Request you to please forward the Monday meeting.